target/linux/realtek/dts-5.10/rtl838x.dtsi

```diff
@@ -148,7 +148,7 @@
 		};
 
 		gpio0: gpio-controller@3500 {
-			compatible = "realtek,rtl8380-gpio", "realtek,otto-gpio";
+			compatible = "realtek,otto-gpio";
```

Comment:
This change is a downgrade, you need both compatibles.

If you only leave the "realtek,otto-gpio" compatible, you lose GPIO interrupts, which is useful for efficiently registering button presses.